Main.java

```diff
@@ -0,0 +1,128 @@
+                break;
+            }
+            int option = scanner.nextInt();
+            switch (option){
+                case 1 -> {
+                    List<String> resultList = listOfName(listOfMaps);
+                    for(int i = 0; i < resultList.size(); i++){
+                        System.out.println(resultList.get(i));
+                    }
+                }
+                case 2 -> {
+                    Double result = sumOfAge(listOfMaps);
+                    System.out.println(result);
+                }
+            }
+        }
+
+    }
+
+    public static List<String> listOfName(List<Map<String, Object>> student) {
+        List<String> listOfStudent = new ArrayList<>();
+        int startIndex = 0;
+        String CandieKey = "candies";
+        String NameKey = "name";
+        for (int i = startIndex; i < student.size(); i++) {
+            for (Entry<String, Object> index : student.get(i).entrySet()) {
+                if (index.getKey().equals(CandieKey)) {
+                    int minCandieAmount = 4;
+                    if ((int) index.getValue() > minCandieAmount) {
+                        String studentNames = (String) student.get(i).get(NameKey);
+                        for (Entry<String, Object> entry : student.get(i).entrySet()) {
+                            if (entry.getValue() == studentNames) {
+                                listOfStudent.add((String) entry.getValue());
+                            }
+                        }
+                    }
+                }
+            }
+        }
+        return listOfStudent;
+    }
+
+    public static double sumOfAge(List<Map<String, Object>> students) {
+        double sum = 0;
+        int indexStart = 0;
+        String CandieKey = "candies";
+        String AgeKey = "name";
+        int maxCandieCount = 5;
+        for (int i = indexStart; i < students.size(); i++) {
+            for (Entry<String, Object> j : students.get(i).entrySet()) {
+                if (j.getKey().equals(CandieKey)) {
+                    if (toDouble(toString(j.getValue())) < maxCandieCount) {
+                        Double studentAge = (Double) students.get(i).get(AgeKey);
+                        for(Entry<String, Object> entry: students.get(i).entrySet()) {
+                            if (entry.getValue() == studentAge) {
+                                sum = sum + objectToDouble(entry.getValue());
+                            }
+                        }
+                    }
+                }
+            }
+        }
+        return sum;
+    }
+    public static String toString(Object convertToString){
+        String something = String.valueOf(convertToString);
+
+        return something;
+    }
+    public static Double toDouble(String convertToDouble){
+        Double something = Double.parseDouble(convertToDouble);
+
+        return something;
+    }
+    public static Double objectToDouble(Object convertObjectToDouble){
```

Comment:
return type and return variable type not match

Reply:
b031e039